tams_ur5_setup_bringup/config/apriltags2/tags.yaml

```diff
@@ -0,0 +1,49 @@
+# # Definitions of tags to detect
+#
+# ## General remarks
+#
+# - All length in meters
+# - Ellipsis (...) signifies that the previous element can be repeated multiple times.
+#
+# ## Standalone tag definitions
+# ### Remarks
+#
+# - name is optional
+#
+# ### Syntax
+#
+# standalone_tags:
+#   [
+#     {id: ID, size: SIZE, name: NAME},
+#     ...
+#   ]
+standalone_tags:
+    [
+    {id: 0, size: 0.161},
+    {id: 352, size: 0.127},
```

Comment:
What is this tag?
If that's your box I believe it should not be in here but in your own package, possibly running in a different instance of the apriltags2_ros node.
Alternatively you might be able to "sneak it in" there in your own launch files via `<param/>`

Please add comments above each tag that say what they refer to.

Reply:
You're right, it's the box! I will look into changing these to optional launch parameters.